File: 11_November_Batch/C/Array/maxInArray.c
#include <stdio.h>
int arrayMax(int n, int arr[])
{
    int max = arr[0], i;
    for (i = 0; i < n; i++)
    {
        if (arr[i]>max)
        {
            max=arr[i];
        }
        
    }
    return max;
}
int main()
{
    int n, ans;
    scanf("%d", &n);
    int arr[n];
    for (int i = 0; i < n; i++)
    {
        scanf("%d", &arr[i]);
    }
    ans = arrayMax(n, arr);

    printf("Max: %d ", ans);

    return 0;
}

File: 11_November_Batch/Python/Array/maxInArray.py
def maximum(n,arr):
    max=arr[0]
    for i in range(1,n):
        if arr[i]>max:
            max=arr[i]
    return max

n=int(input())
arr=list(map(int,input().split()))
ans=maximum(n,arr)
print(ans)